docs/src/classes.md

```diff
@@ -0,0 +1,33 @@
+```@meta
+CurrentModule = GenericCharacterTables
+```
+
+# Conjugacy class types
+
+TODO: recall what those are
+
+## Properties
+
+```@docs
+classtypes
+nrclasses
+centord
+printclassparam
+printinfoclass
+```
+
+## Norms and scalar products
+
+`GenericCharacterTables` allows you to compute norms and scalar products
+of class types. The results are correct for all
+possible combinations of parameters except possibly for those where the
+additionally returned exceptions apply. Those consist of multivariate
+polynomials with coefficients in a rational function field and are
+satisfied if the evaluation of this polynomial is an integer.
+
+
+```@docs
+ortho2norm
+ortho2scalar
```

Comment:
Should we rename those to norm resp scalar_product ?

Reply:
Yes, that's a good idea. As we don't have the old index based methods for the characters anymore we should get any conflicts.